doc/content/getting-started/se-hands-on-guide/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "SENSOR EDITION HANDS ON TUTORIALS"
```

Comment:
Don't use all caps here, use sentence case and we can add some CSS to capitalize it if we want to later

Reply:
Done